lib/draw.c

```diff
@@ -178,6 +265,8 @@ void projectMeshTo2D(mesh inputMesh, const double distance)
         {
         tempVec = inputMesh.tris[i].p[j];
 
+        tempVec.x = tempVec.x * CHAR_CONST;
```

Comment:
if this char_const is purely about scaling for your ascii display, then I think this scaling logic should go in the draw function. your 2d projection shouldn't know or care about the output medium